src/apps/loyalty/containers/login/__tests__/__snapshots__/index.tsx.snap

```diff
@@ -64,7 +64,7 @@ exports[`login renders the snapshot 1`] = `
       className="eTVvmy eYaNZQ"
       href="/twitter"
       icon={
-        <Styled(Icon)
+        <Styled(Component)
```

Comment:
Hmm, this fails on CI. I thought it was weird that these snapshots didn’t match on my machine anyways, so must be something else. I’ll revert this so the PR is mergable and look into why it fails on my machine afterwards.